### PR TITLE
Created Public Folder

### PR DESCRIPTION
**Pull request by pabaross**
Public Source

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      This HTML file is a template.
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
```
Comment (yuukiross) [inline frontend/public/index.html:41]: remove here

File: frontend/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
```
Comment (yuukiross) [inline frontend/public/index.html:26]: remove here

File: frontend/public/index.html (hunk 1 above)
Comment (pabaross) [reply frontend/public/index.html]: ok

File: frontend/public/index.html (hunk 2 above)
Comment (pabaross) [reply frontend/public/index.html]: ok

**Review by penguin011 — COMMENTED** [review]
👍 good

**Pull request merged**